marconi-chain-index/test/Spec/Marconi/ChainIndex/Indexers/Utxo/UtxoIndex.hs

```diff
@@ -312,141 +326,157 @@ propSaveAndRetrieveUtxoEvents = property $ do
-propCompareRetrieveUtxoEventsFromInMemoryAndSQLite :: Property
-propCompareRetrieveUtxoEventsFromInMemoryAndSQLite = property $ do
-  highSlotNo <- forAll $ Gen.integral $ Range.constantFrom 7 5 20
-  chainPoints :: [C.ChainPoint]  <- forAll $ genChainPoints 2 highSlotNo
-  events::[StorableEvent Utxo.UtxoHandle] <- forAll $ forM chainPoints genEventWithShelleyAddressAtChainPoint <&> concat
-  -- depth <- forAll $ Gen.int (Range.constantFrom (numOfEvents - 1) 1 (numOfEvents + 1))
-  indexerMixed <- liftIO . raiseException $ Utxo.open ":memory:" (Utxo.Depth 1) False -- don't vacuum sqlite
-             >>= Storable.insertMany events
-  indexerMemory <- liftIO . raiseException $ Utxo.open ":memory:" (Utxo.Depth 2160) False -- don't vacuum sqlite
-             >>= Storable.insertMany events
-  let lastSlot = getSlot $ last chainPoints
-      qAddressesAtSlot
-        = List.nub  -- remove duplicate addresses
-        . fmap (flip Utxo.UtxoByAddress lastSlot . Utxo._address)
-        . concatMap (Set.toList . Utxo.ueUtxos)
-        $ events
-      qAddressesAll
-        = List.nub  -- remove duplicate addresses
-        . fmap (flip Utxo.UtxoByAddress Nothing . Utxo._address)
-        . concatMap (Set.toList . Utxo.ueUtxos)
-        $ events
-  -- resultAtSlot <- liftIO . traverse (Storable.query Storable.QEverything indexer) $ qAddressesAtSlot
-  qAllresultMixed :: Set Utxo.UtxoRow <-
-    liftIO . fmap getRows . raiseException . traverse (Storable.query Storable.QEverything indexerMixed) $ qAddressesAll
-  qAllresultMemory :: Set Utxo.UtxoRow <-
-    liftIO . fmap getRows . raiseException . traverse (Storable.query Storable.QEverything indexerMemory) $ qAddressesAll
-  qSlotresultMixed :: Set Utxo.UtxoRow <-
-    liftIO . fmap getRows . raiseException . traverse (Storable.query Storable.QEverything indexerMixed) $ qAddressesAtSlot
-  qSlotresultMemory :: Set Utxo.UtxoRow <-
-    liftIO . fmap getRows . raiseException . traverse (Storable.query Storable.QEverything indexerMemory) $ qAddressesAtSlot
-
-  Hedgehog.assert $ (not . null) qAllresultMixed
-  qAllresultMixed === qAllresultMemory
-
-  Hedgehog.assert $ (not . null) qSlotresultMixed
-  length qSlotresultMixed === length qSlotresultMemory
-  qSlotresultMixed === qSlotresultMemory
-
--- Insert Utxo events in storage, and retreive the events by address and slot
--- Note: The property we are checking is:
---   - Insert many events at various chainPoints
---   - Fetch for all the evenent addresses
---   - There should not be any results with ChainPoint > highChainPoint
-propUtxoQueryByAddressAndSlot :: Property
-propUtxoQueryByAddressAndSlot = property $ do
+ ---------------------------------------------------------------------
+-- TODO --
+-- Uncommenting the TODO section below will fail the test, see ticket PLT-5937
+-- Add this to observe the Integer Overflow in the logs:
+-- liftIO $ SQL.setTrace (Utxo.hdlConnection (h ^. Storable.handle)) $ Just (Text.IO.appendFile "utxoSQLiteTrace.loger")
```

Comment:
@berewt  adding this line to right after the storage `open`  sill log the SQL statements.